src/features/movies/MoviePage/MoviePoster/styled.js

```diff
@@ -54,6 +55,10 @@ export const Image = styled.img`
     max-width: 72vw;
     aspect-ratio: 16/9;
     z-index: -1;
+    background-image: url(${noresult});
```

Comment:
we decided to not render this component on daily today

Reply:
Adding `background-color: black;` (of course black taken from theme.js) here is even better because until the image loads we should rather see a black background instead of no results. Ultimately we should not render this component at all if backdrop_path from API is null, it also gives 404 error in console now.

Reply:
i made that but it was not commitet hah
done